Snippets/Snippets_6/Lifecycle/EndpointStartAndStop.cs

```diff
@@ -2,19 +2,48 @@
 {
     using System.Threading.Tasks;
     using NServiceBus;
+    using NServiceBus.Features;
 
     #region lifecycle-EndpointStartAndStop
 
-    class RunWhenEndpointStartsAndStops : IWantToRunWhenEndpointStartsAndStops
+    class RunWhenEndpointStartsAndStops
```

Comment:
We just removed the FeatureStartupTask part. We will just point that to the article with it's existing snippets